workflow/rocoto/gefs_tasks.py

```diff
@@ -339,11 +340,24 @@ def wavepostsbs(self):
                      'maxtries': '&MAXTRIES;'
                      }
 
-        member_var_dict = {'member': ' '.join([str(mem).zfill(3) for mem in range(0, self.nmem + 1)])}
-        member_metatask_dict = {'task_name': 'gefs_wave_post_grid',
-                                'task_dict': task_dict,
-                                'var_dict': member_var_dict
-                                }
+        fhrs = self._get_forecast_hours('gefs', self._configs['wavepostsbs'], 'wave')
+
+        is_replay = self._configs['wavepostsbs']['REPLAY_ICS']
+        if is_replay:
+            for hour in [0, 1, 2]:
+                if hour in fhrs:
+                    fhrs.remove(hour)
```

Comment:
This is a more pythonic way of accomplishing the same thing:
```suggestion
        if is_replay:
            fhrs = [ fhr in fhrs if fhr not in [0, 1, 2] ]
```